### PR TITLE
Add HimerosTV & TimTales xPath scene scraper

### PR DESCRIPTION
**Pull request by whitty-username**
HimerosTV image scrape doesn't appear to be working for some reason. Might need some help on that one.

TimTales appears to work as expected.

### CODE REVIEW

File: scrapers/HimerosTV.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      Tags:
+        Name: 
+          selector: //div[@class='scene-tags']//ul//a
+      Image:
+        selector: //div[@class='mejs-poster mejs-layer']//img/@src
+        replace:
+          - regex: ^
+            with: "https://himeros.tv"
+          - regex: "-1x.jpg"
+            with: "-3x.jpg"
```
Comment (Belleyy) [inline scrapers/HimerosTV.yml:33]: Forget to use PostProcess.

```    
        postProcess:
          - replace:
              - regex: ^
                with: "https://himeros.tv"
              - regex: "-1x.jpg"
                with: "-3x.jpg"
```

File: scrapers/TimTales.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        Name: 
+          selector: //div[@class='text']//p[@class='categories'][contains(text(),'Men:')]/a
+      Tags:
+        Name: 
+          selector: //div[@class='text']//p[@class='categories'][contains(text(),'Categories:')]/a
+      Image:
+        selector: //*[@class='video player no-volume play-button']/@style
+        replace:
+          - regex: .+(https[^')]+).+
+            with: $1
```
Comment (Belleyy) [inline scrapers/TimTales.yml:31]: Same as above, PostProcess for the replace

**Comment by Belleyy** [comment]
- You forget to edit the SCENE-SCRAPABLE.md
- You can add at the end of file `#Last Updated August 20, 2020`
- HimerosTV scraper don't seem to work for me (not only the image), can you give a link maybe i take a wrong one ? (`https://himeros.tv/tour/trailers/Dream-Dick.html`)

**Comment by whitty-username** [comment]
@Belleyy Apologies, I'm a total noob so forgive me for those oversights. It looks like HimerosTV was coincidentally updated **literally** within the last 24 hours. I'll try and update the scraper as its broken now on my end too.

**Comment by whitty-username** [comment]
Okay so HimerosTV is fixed now, but still having trouble with the image. Essentially the domain needs to be injected at the start of the string to complete the image URL. Not sure if I'm using the right command for that. 🤷🏻‍♂️

**Comment by Belleyy** [comment]
For the image : `//div[@class='player-thumb']/img[@class="update_thumb thumbs stdimage"]/@src0_1x`

Why? Because your current selector appear after the page is loaded (javascript change thing). 
When creating scraper, try disabling javascript (if you use Ublock you can block it) then you find the XPath.

Other solution would be to use CDP. Scraper would wait the page to load then getting the XPath  (Check CDP support in Wiki for more info)
```
driver:
  useCDP: true
```

**Comment by whitty-username** [comment]
Took me a while but I figured it out 😂

**Pull request merged**